web-module/src/main/webapp/resources/celJS/images/imageGalleryPaging.js

```diff
@@ -352,8 +352,12 @@
       var htmlElems = _me._curContentObj.content;
       var listElementsArr = null;
       if ((htmlElems != '') && (htmlElems != null)) {
-        var listElements = new Element('div').update(htmlElems).down('ul').children;
-        var listElementsArr = Array.prototype.slice.call(listElements);
+        var elem = new Element('div').update(htmlElems).down('ul');
+        var listElements = new Array();
+        if(!elem.hasClassName('cel_nav_empty')) {
+          var listElements = new Element('div').update(htmlElems).down('ul').children;
+        }
+        var listElementsArr = Array.prototype.slice.call(listElements);          
```

Comment:
is there a reason for not calling listElements.slice() directly? And what is this call for?